centralized-sampling-tests/sample-apps/spring-boot/src/main/java/com/amazon/sampleapp/Application.java

```diff
@@ -0,0 +1,70 @@
+package com.amazon.sampleapp;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.trace.propagation.W3CTraceContextPropagator;
+import io.opentelemetry.context.propagation.ContextPropagators;
+import io.opentelemetry.context.propagation.TextMapPropagator;
+import io.opentelemetry.contrib.awsxray.AwsXrayIdGenerator;
+import io.opentelemetry.contrib.awsxray.AwsXrayRemoteSampler;
+import io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter;
+import io.opentelemetry.extension.aws.AwsXrayPropagator;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.sdk.trace.export.BatchSpanProcessor;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+public class Application {
+
+	public static Resource resource = Resource.builder().build();
+
+	// Opentelemetry builder to create a xray remote sampler with polling interval of 1
+	public static OpenTelemetry openTelemetry =
```

Comment:
If openTelemetry is not static, we can new a opentelemetry or tracer instead of rebooting sample app?

Reply:
yes but we shouldn't ever need to reboot the sample-app based on current design